.github/workflows/test.yml

```diff
@@ -58,9 +41,4 @@ jobs:
 
       - name: "Test package"
         # prettier-ignore
-        run: "python -m pytest -ra --durations=20 --cov --cov-report=xml --cov-report=term tests/unit"
-
-      - name: "Upload coverage report"
```

Comment:
I don't think this was ever working, although it did not raise an error. This was something included in the scientific python template we used to initialize this repo, and since we have never used it, no problem with removing!